handler/instance_started_test.go

```diff
@@ -61,23 +61,38 @@ type statusCoder interface {
 	StatusCode() int
 }
 
+func testCfg() config.Config {
+	return config.Config{
+		KafkaConfig: config.KafkaConfig{
+			Addr:                         []string{"localhost:9092", "localhost:9093"},
+			InstanceStartedGroup:         "dp-import-cantabular-dataset",
+			InstanceStartedTopic:         "cantabular-dataset-instance-started",
+			CategoryDimensionImportTopic: "cantabular-dataset-category-dimension-import",
+		},
+	}
+}
+
 func TestInstanceStartedHandler_HandleHappy(t *testing.T) {
-	cfg := config.Config{}
+	testCfg := testCfg()
 
 	Convey("Given a successful event handler", t, func() {
 		ctblrClient := cantabularClientHappy()
 		recipeAPIClient := recipeAPIClientHappy()
 		importAPIClient := importAPIClientHappy()
 		datasetAPIClient := datasetAPIClientHappy()
-		producer := kafkatest.NewMessageProducer(true)
+		producer, _ := kafkatest.NewProducer(ctx, &kafka.ProducerConfig{
```

Comment:
should add an err and `So(err, ShouldBeNil)`
(Same for a number of these new changes later in this file)

Reply:
These have now been added.